### DISCUSSION TITLE
All Unavailable

### DISCUSSION THREAD

**Discussion by MarkGTI** [TheHolyRoger/hass-cryptoinfo#6] [General]
Hi all

Installed a few days ago. Ran great for 24 hours and now everything is unavailable. Any ideas?

**Comment by radeon79** [comment]
I installed and configured it yesterday too, it works but every time there's an update the sensors go into a 'not available' state for a few seconds, distorting all the data. This happens regardless of the seconds I set for the update. In this condition, it's not usable. Are these API limitations or is there some other issue?
